src/System.Diagnostics.Process/tests/Configurations.props

```diff
@@ -4,6 +4,7 @@
     <BuildConfigurations>
       netstandard-Windows_NT;
       netstandard-Unix;
+      uap-Windows_NT;
```

Comment:
talked offline with @krwq and instead of adding uap configuration, we can differentiate the test cases according to OSGroup instead and SkipOnFrameworkTarget attributes